### ISSUE TITLE
x16r on zergpool benchmarking failed?

### ISSUE THREAD

**Issue by joeyoon80** [Sniffdog/Sniffdogminer#178]
Hi. I am facing an issue with latest 4.8.2 version. Attached is the pic for reference:

![image](https://user-images.githubusercontent.com/32901879/37217538-c46f8f68-23f8-11e8-8e7a-814917ea80a1.png)

**Comment by Sniffdog** [comment]
yeah you may have to remove those optimizations in CcminerRavenEnemy.ps1

![screenshot 53 _li](https://user-images.githubusercontent.com/30452757/37222047-1bd08ffc-2389-11e8-9574-cec0d34cb50e.jpg)

**Comment by Sniffdog** [comment]
those optimizations have been removed and updated at release

**Comment by joeyoon80** [comment]
Thanks. Removing the optimizations didnt work, so I turned on x16r with Tpruvot Raven.

I have another question. In my bat file I set to mine a few algos for zergpool. I also watched them benchmark properly. However, they are not listed on the mining monitoring after benchmark completed.

![image](https://user-images.githubusercontent.com/32901879/37240358-1e503ad2-2485-11e8-821e-0e5410dc4264.png)

![image](https://user-images.githubusercontent.com/32901879/37240364-306543f2-2485-11e8-8b59-50f465892133.png)

**Comment by Sniffdog** [comment]
awesome glad you got sniffdog running

x17 was also a enemy algo so if you want you can goto ccmineralexis.ps1 and edit file in Miners folder and remove the # infront of the algo x17

**Comment by syrm** [comment]
Still don't work for me.
Bin\NVIDIA-Enemy\ccminer.exe -a x16r etc...
"Error: no algo or invalid algo"
